.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
```

Comment:
I assume these are used to prefill the issue fields? Not too familiar with the github template system.

Reply:
Maybe leave it empty then? I'm also not familiar with github issue template thing

Reply:
Let's see what happens if we merge this!